.github/workflows/nightly.yml

```diff
@@ -0,0 +1,32 @@
+name: Nightly Test Suite
+
+on:
+  schedule:
+    - cron: 0 0 * * * # daily at midnight
+  workflow_dispatch: # or manually dispatch the job
+
+jobs:
+  nightly-test-suite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Quick Install Nix
+        uses: cachix/install-nix-action@v22
+        with:
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+            accept-flake-config = true
+
+      - name: plutus-core-nightly
+        if: always()
+        run: |
+          nix run --no-warn-dirty --accept-flake-config \
```

Comment:
You need to be in the package directory for this. You would probably want to build the derivation in `checks` if we were doing it this way.